### DISCUSSION TITLE
coding place

### DISCUSSION THREAD

**Discussion by Zuhair1706** [teamrudra/R24-Test-Hex-Editor#13] [Q&A]
where are we supposed to do coding to get the solution?

**Comment by waseemR02** [comment]
> where are we supposed to do coding to get the solution?
> 

You need to fork the repo to your account and then clone it. Once you do that, you will have a copy on your local setup and then you can make changes to it . 
Make sure you make some changes to the repo and commit them. Push it to your repo and you will get an option to raise a PR.
